tests/client/ui/components/TestRating.js

```diff
@@ -92,6 +98,33 @@ describe('ui/components/Rating', () => {
     });
   });
 
+  it('renders selected stars nearest to an average rating', () => {
```

Comment:
This description confuses me a bit, I feel like it's saying "should round down average ratings to stars" or something, as the crux of the test seems to be that we don't round up.